core/src/apps/benfen/sign_tx.py

```
@@ -0,0 +1,235 @@
+from .helper import INTENT_BYTES, benfen_address_from_pubkey, try_convert_to_bfc_address
+from .layout import require_confirm_fee, require_show_overview
+from .tx_parser import TransactionParser
+
+
+async def process_transaction(
+    ctx, address: str, tx_bytes: bytes, coin_type: bytes
+) -> bytes:
+    parser = TransactionParser()
+
+    intent = tx_bytes[:3]
+    if INTENT_BYTES != intent:
```

Ruff (0.8.2)

Yoda condition detected

Rewrite as `intent != INTENT_BYTES`

(SIM300)